shepherd_bms/src/compute.cpp

```diff
@@ -282,3 +282,47 @@ void ComputeInterface::MCCallback(const CAN_message_t &msg)
 {
     return;
 }
+
+uint8_t ComputeInterface::calcChargerLEDState(uint8_t state_of_charge, AccumulatorData_t *bms_data)
```

Comment:
As mentioned before rebase, I'd like to see SoC added to bms data and then just take the bms data for this function, not adding SoC as an extra param

Reply:
yeah plan was to still do this before merge, nick just wanted to rebase first. If okay with @nwdepatie ill check this branch out and push these last few changes in the am